tests/unit/components/pacbio/PacbioRunWellEdit.spec.js

```diff
@@ -18,9 +18,10 @@ const smrtLinkVersions = {
 }
 
 const propsData = {
-  position: 'A1',
   isStatic: true,
 }
+const position = 'A1'
```

Comment:
Check `PacbioRunWellEdit` props vs arg for attrs

Reply:
@harrietc52

Reply:
I think the way it is currently implemeneted (with position and plateNumber as arg into a method) is best.